exercises/luhn/example.rs

```diff
@@ -1,5 +1,6 @@
 pub fn is_valid(candidate: &str) -> bool {
-    if candidate.chars().any(|c| c.is_alphabetic()) || candidate.chars().count() == 1 {
+    if input.chars().filter(|c| c.is_digit(10)).take(2).count() <= 1 ||
```

Comment:
you seem to have intended to change the input's name to `input`, but didn't make the same change in the argument. I believe if you were to do that, the tests could pass.